openday_scavenger/static/html/404_invalid_uid.html

```diff
@@ -1,9 +1,17 @@
 {% extends "layout.html" %}
 {% block content %}
-<div class="d-flex justify-content-center align-items-center">
-    <div class="col-md-12 text-center">
-        <h1>404</h1>
-        <h2>Invalid UID</h2>
-    </div>
+<div class="container my-4 text-center">
+    <h1 class="type-h1 mb-3">⚠️ Oh no! ⚠️</h1>
+    <p>
+        👾 <b>You've found a 404 invalid UID error</b>! 👾<br />
+        Have you registered for the Scavenger Hunt?
+    </p>
+    <p class="mt-5">
+        If you'd like to join the scavenger hunt,<br />
+        come to Scientific Computing booth!
```

Comment:
"come to Scientific Computing booth" -> "come to the Scientific Computing booth"

Reply:
Thank you for catching that!! Haha, updated!